<launch>
  <arg name="input_topic" default="points_raw"/>
  <arg name="output_topic" default="filtered_points"/>
  <arg name="voxel_leaf_size" default="1.0"/>

  <node pkg="voxel_grid_filter"
        name="voxel_grid_filter_node"
        type="voxel_grid_filter_node"
        output="screen">
    <param name="input_topic" value="$(arg input_topic)"/>
    <param name="output_topic" value="$(arg output_topic)"/>
    <param name="voxel_leaf_size" value="$(arg voxel_leaf_size)"/>
  </node>
</launch>
